GoogleForCreators/web-stories-wp
# Fix highlight issues

## Summary

I wanted to fix #11288 and #11036 (and maybe other related).
Some of the highlight issues seem new, but they did not work for a long time (I checked on 1.15).
This change seems to fix all issues - I have tested all combinations and everything worked as expected.

## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Use quick action to trigger the highlight.
2. Check if it works.
3. Check all combinations (before/after the first highlight change a section/tab etc.).


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #11288
Fixes #11036
